compose/gettingstarted.md

```diff
@@ -98,73 +96,142 @@ This Compose file defines two services, `web` and `redis`. The web service:
+    Creating network "composetest_default" with the default driver
+    Creating composetest_web_1 ...
+    Creating composetest_redis_1 ...
+    Creating composetest_web_1
+    Creating composetest_redis_1 ... done
+    Attaching to composetest_web_1, composetest_redis_1
+    web_1    |  * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
+    redis_1  | 1:C 17 Aug 22:11:10.480 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
+    redis_1  | 1:C 17 Aug 22:11:10.480 # Redis version=4.0.1, bits=64, commit=00000000, modified=0, pid=1, just started
+    redis_1  | 1:C 17 Aug 22:11:10.480 # Warning: no config file specified, using the default config. In order to specify a config file use redis-server /path/to/redis.conf
+    web_1    |  * Restarting with stat
+    redis_1  | 1:M 17 Aug 22:11:10.483 * Running mode=standalone, port=6379.
+    redis_1  | 1:M 17 Aug 22:11:10.483 # WARNING: The TCP backlog setting of 511 cannot be enforced because /proc/sys/net/core/somaxconn is set to the lower value of 128.
+    web_1    |  * Debugger is active!
+    redis_1  | 1:M 17 Aug 22:11:10.483 # Server initialized
+    redis_1  | 1:M 17 Aug 22:11:10.483 # WARNING you have Transparent Huge Pages (THP) support enabled in your kernel. This will create latency and memory usage issues with Redis. To fix this issue run the command 'echo never > /sys/kernel/mm/transparent_hugepage/enabled' as root, and add it to your /etc/rc.local in order to retain the setting after a reboot. Redis must be restarted after THP is disabled.
+    web_1    |  * Debugger PIN: 330-787-903
+    redis_1  | 1:M 17 Aug 22:11:10.483 * Ready to accept connections
+    ```
+
+    Compose pulls a Redis image, builds an image for your code, and start the
+    services you defined. In this case, the code is statically copied into the image at build time.
+
+2.  Enter `http://0.0.0.0:5000/` in a browser to see the application running.
+
+    If you're using Docker on Linux natively, then the web app should
+    now be listening on port 5000 on your Docker daemon host.
+    If `http://0.0.0.0:5000` doesn't resolve, you can also
+    try `http://localhost:5000`.
+
+    If you're using Docker Machine on a Mac, use  
+    `docker-machine ip MACHINE_VM` to get the IP address of your
+    Docker host. Then, open `http://MACHINE_VM_IP:5000` in a browser.
+
+    You should see a message in your browser saying:
+    ```
+    Hello World! I have been seen 1 times.
+    ```
+
+3.  Refresh the page.
+
+    The number should increment.
+
+    ```
+    Hello World! I have been seen 2 times.
+    ```
 
-        $ docker-compose up
-        Pulling image redis...
-        Building web...
-        Starting composetest_redis_1...
-        Starting composetest_web_1...
-        redis_1 | [8] 02 Jan 18:43:35.576 # Server started, Redis version 2.8.3
-        web_1   |  * Running on http://0.0.0.0:5000/
-        web_1   |  * Restarting with stat
+4.  Switch to another terminal window, and type `docker image ls` to
```

Comment:
@joaofnfernandes it won't run it in the background without the `-d` option, which we show them later. A few of the tutorials are written like this, maybe I'll have a look and introduce `-d` sooner than was originally done.